into/backends/csv.py

```diff
@@ -237,13 +239,20 @@ def _():
     return chunks(type(r))(_)
 
 
-@convert.register(chunks(pd.DataFrame), chunks(CSV), cost=10.0)
+@convert.register(chunks(pd.DataFrame), (chunks(CSV), chunks(Temp(CSV))), cost=10.0)
 def convert_glob_of_csvs_to_chunks_of_dataframes(csvs, **kwargs):
     def _():
         return concat(convert(chunks(pd.DataFrame), csv, **kwargs) for csv in csvs)
     return chunks(pd.DataFrame)(_)
 
 
+@convert.register(Temp(CSV), (pd.DataFrame, chunks(pd.DataFrame)))
+def convert_dataframes_to_temporary_csv(data, **kwargs):
+    fn = '.%s.csv' % uuid.uuid1()
```

Comment:
I'd like to punt for the moment on gzip.  I'm inclined to use a `compression=` kwarg.

Actually, thanks to us passing all kwargs everywhere this might have already worked as is.  Sadly we don't send `**kwargs` to `DataFrame.to_csv` and, even worse, `DataFrame.to_csv` doesn't seem to support gzip writing.